UI/Pages/Shortlist.cshtml.cs

```diff
@@ -158,7 +163,7 @@ private string[] GetShortlistSeoUrls()
             if (location is IErrorResult error)
             {
                 //Shouldn't be invalid, unless query string edited - since postcode on this page comes from previous page with validation
-                _logger.LogWarning("Invalid postcode '{Postcode}' provided on shortlist page", request.Postcode);
+                _logger.LogInformation("Invalid postcode '{Postcode}' provided on shortlist page", request.Postcode);
```

Comment:
Guess that the validation kicks in before this?  So should never get here now anyway?  If that's the case, maybe leave it as a warning, so we see the log if it does somehow get to this point?

Reply:
Yeah, good point. With the new validation logic in place, we validate input before calling the `Data = await _mediator.Send(data);` therefore, we can safely revert that logic to its previous log level, which is a warning.